FAQ.md

```diff
@@ -5,6 +5,8 @@
 Make sure to install ansible and clone the bbb-configs repository. Then install the requirements using:
 
 ```sh
+apt update
+apt install -y jq
```

Comment:
If we go that route we should also mention to install the general build prerequisites: https://openwrt.org/docs/guide-developer/toolchain/install-buildsystem

Reply:
at least `make` will be missing. I ran into this yesterday, when I setup a new VM to build images based on latest ubuntu server LTS.

Reply:
Or just add another sentence that there might be more requirements depending on the system you are building on and if builds fail you might want to check out that OpenWRT page.